File: QuanLySanCauLong/lib/models/user_info.dart
class UserInfo {
  final String email;
  final String sdt;
  final String gioiTinh;
  final String hoTen;

  UserInfo({
    required this.email,
    required this.sdt,
    required this.gioiTinh,
    required this.hoTen,
  });

  factory UserInfo.fromJson(Map<String, dynamic> json) {
    return UserInfo(
      email: json['email'],
      sdt: json['sdt'],
      gioiTinh: json['gioiTinh'],
      hoTen : json['hoTen']
    );
  }
}

File: quan_ly_san_cau_long_API/quan_ly_san_cau_long_API/Models/UserInfo.cs
namespace quan_ly_san_cau_long_API.Models
{
    public class UserInfo
    {
        public int Id { get; set; }
        public string HoTen { get; set; }
        public string Email { get; set; }
        public string SDT { get; set; }
        public string GioiTinh { get; set; }

        public int UserId { get; set; }
        public User User { get; set; }
    }
}
